packages/block-editor/src/components/global-styles/filters-panel.js

```diff
@@ -75,6 +90,36 @@ const DEFAULT_CONTROLS = {
 	duotone: true,
 };
 
+const popoverProps = {
+	placement: 'left-start',
+	offset: 36,
+	shift: true,
+	className: 'block-editor-duotone-control__popover',
+	headerTitle: __( 'Duotone' ),
+};
+
+const LabeledColorIndicators = ( { indicators, label } ) => (
```

Comment:
Why is this receiving an array of indicators, it seems we always use it with a single one? This component might be simplified.